spring-integration-debezium/src/main/java/org/springframework/integration/debezium/inbound/support/DebeziumHeaders.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2023-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.debezium.inbound.support;
```

Comment:
I think you have missed something.
These headers-related classes must be in the `org.springframework.integration.debezium.support` package - top-level of the module, not a nested one in still that `inbound`.